File: DNAStrand.js
function DNAStrand(dna){
  let newStr = ''
  let strand = new Map()
  strand.set("C", "G")
  strand.set("G", "C")
  strand.set("A", "T")
  strand.set("T", "A")
  
  //your code here
  let letters = dna.split('')
  for (let letter of letters) {
    while (dna.includes(letter)) {
           newStr = dna.replace(letter, strand.get(letter))
           })
  }
  return newStr
}

File: DNAStrand.py
def DNA_strand(dna):
    strands = {'A': 'T',
               'T': 'A',              
               'C': 'G',
               'G': 'C' }  
              
    # code here
    for letter in dna:
        dna = dna.replace(letter, strands.get(letter))
    return dna